README.rst

```diff
@@ -389,7 +389,7 @@ If you have multiple requirement files make sure you create a hook for each file
             files: ^requirements-lint\.(in|txt)$
           - id: pip-compile
             name: pip-compile requirements.txt
-            args: [requirements.txt]
+            args: [requirements.in]
```

Comment:
```suggestion
```

`requirements.in` is the default input file, so there is no need to pass the filename.